### PR TITLE
Change filters contin list name

### PR DESCRIPTION
**Pull request by etiennehomer**

### CODE REVIEW

File: src/main/java/org/gridsuite/actions/server/ContingencyListController.java [hunk 1]
```diff
@@ -116,22 +116,22 @@ public ContingencyListController(ContingencyListService service) {
...
     }
 
-    @PutMapping(value = "/filters-contingency-lists/{id}", consumes = MediaType.APPLICATION_JSON_VALUE)
-    @Operation(summary = "Modify a filters contingency list")
-    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The filters contingency list have been modified successfully")})
-    public ResponseEntity<Void> modifyFilterContingencyList(@PathVariable UUID id, @RequestBody(required = true) FiltersContingencyListAttributes filtersContingencyListAttributes) {
+    @PutMapping(value = "/form-contingency-lists/{id}", consumes = MediaType.APPLICATION_JSON_VALUE)
+    @Operation(summary = "Modify a form contingency list")
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The form contingency list have been modified successfully")})
+    public ResponseEntity<Void> modifyFormContingencyList(@PathVariable UUID id, @RequestBody(required = true) FormContingencyList formContingencyList) {
```
Comment (AbdelHedhili) [inline src/main/java/org/gridsuite/actions/server/ContingencyListController.java:132]: Not sure what is best here, should we keep the attributes class or not... 
@niconoir what do you think ?

**Review by AbdelHedhili — COMMENTED** [review]
Everything seems ok for me. Just need confirmation that it's ok to remove the attribute class

File: src/main/java/org/gridsuite/actions/server/ContingencyListService.java [hunk 2]
```diff
@@ -50,37 +50,37 @@
...
-    private static FiltersContingencyList fromFilterContingencyListEntity(FiltersContingencyListEntity entity) {
-        return new FiltersContingencyList(entity.getId(), entity.getEquipmentId(), entity.getEquipmentName(),
+    private static FormContingencyList fromFormContingencyListEntity(FormContingencyListEntity entity) {
+        return new FormContingencyList(entity.getId(), entity.getEquipmentId(), entity.getEquipmentName(),
             entity.getEquipmentType(), entity.getNominalVoltage(), entity.getNominalVoltageOperator(), entity.getCountries());
     }
 
-    private static FiltersContingencyListAttributes fromFilterContingencyListEntityAttributes(FiltersContingencyListEntity entity) {
-        return new FiltersContingencyListAttributes(entity.getId(), entity.getEquipmentId(), entity.getEquipmentName(),
+    private static FormContingencyList fromFormContingencyListEntityAttributes(FormContingencyListEntity entity) {
```
Comment (niconoir) [inline src/main/java/org/gridsuite/actions/server/ContingencyListService.java:82]: Methods fromFormContingencyListEntity and fromFormContingencyListEntityAttributes are the same. Remove one of them.

**Review by niconoir — CHANGES_REQUESTED** [review]

File: src/main/java/org/gridsuite/actions/server/ContingencyListController.java (hunk 1 above)
Comment (niconoir) [reply src/main/java/org/gridsuite/actions/server/ContingencyListController.java:132]: I'm not sure it is necessary to keep attributes class here. For script contingency lists creations or modifications, we use a ScriptContingencyList class, so it seems consistent to use FormContingencyList class for form contingency lists creations or modifications too.

File: src/main/java/org/gridsuite/actions/server/ContingencyListService.java (hunk 2 above)
Comment (etiennehomer) [reply src/main/java/org/gridsuite/actions/server/ContingencyListService.java]: Done

**Pull request merged**